Automattic/vip-go-node-skeleton
# Remove babel dependency

This PR aims to remove the `babel` dependency of the Node.js skeleton, and remove the `.babelrc` file. The `build` step has been changed to only output a simple message linking the developer to the documentation, where more information on the build process can be found.

The message is not set in stone, so I'm happy to tweak it if needed.

## How to test

1. Checkout this PR.
2. Run:
```
npm install
npm run build
```
3. You should see `This project does not have a build script configured. For further information, refer to our documentation at https://docs.wpvip.com/technical-references/vip-platform/node-js/#h-requirement-2-production-dependencies-and-npm-scripts` being outputted.
4. Run `npm start` and validate that the application runs.
